api/clients/v2/relay_payload_retriever.go

```diff
@@ -187,12 +174,24 @@ func (pr *RelayPayloadRetriever) GetPayload(
 func (pr *RelayPayloadRetriever) getBlobWithTimeout(
 	ctx context.Context,
 	relayKey core.RelayKey,
-	blobKey *core.BlobKey) ([]byte, error) {
+	blobKey *core.BlobKey,
+	blobLengthSymbols uint32,
```

Comment:
can you add documentation as to where blobLengthSymbols typically comes from when calling this function? Gotten from the cert right?